README.md

```diff
@@ -149,12 +147,10 @@ optional arguments:
   --target-repo TARGET_REPO
                         The target repo if different to the source-repo where
                         data will be restored.
-  -w, --wait            Wait the end of deployment before stop the script.
-                        Raise anexception if deployment failed in the given
-                        time
+  -d, --no-wait         Run the script in detached mode : do not wait the end of deployment to stop the script.
```

Comment:
The shortcut is ``-d`` is cool but I don't feel so natural, in that case I would set any shortcut leaving only the long version ``--no-wait```anyway I won't blok for that one

Reply:
OK! linux daemon mode habits..